openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -37,45 +37,21 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't think this bit is needed: "No additional complications are introduced."</p><p> </p><p>After reading the next paragraph, I think you were referring to MPI operations as "additional complications", but it's not clear here and I think unnecessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Removed</p>